pkg/network/tracer/tracer_test.go

```diff
@@ -1514,12 +1514,11 @@ func TestHTTPSViaOpenSSLIntegration(t *testing.T) {
 		t.Skip("libssl.so not found; skipping test.")
 	}
 
-	os.Setenv("SSL_LIB_PATHS", libSSLPath)
-
 	// Start tracer with HTTPS support
 	cfg := testConfig()
 	cfg.EnableHTTPMonitoring = true
 	cfg.EnableHTTPSMonitoring = true
+	cfg.BPFDebug = true
```

Comment:
Unless this is needed for the test to function, I'd like to leave these out. It makes the tests run slower.

Reply:
👍